src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -110,17 +120,27 @@ public bool IsEmpty
         /// </summary>
         public T[] CreateArray()
```

Comment:
I started to use Create to clearly indicate that a heap allocation is happening. But maybe it's an overkill. I would be fine with ToArray.